docs/UsersGuide.asciidoc

```diff
@@ -1032,6 +1032,23 @@ openqa-dump-templates --json --group test > test.json
 openqa-load-templates test.json
 --------------------------------------------------------------------------------
 
+== Timers and triggers ==
```

Comment:
Alternatively to my other comment, this *could* be a sub-section of the "Cleanup" section under "Advanced configuration". Of course the triggers also work for any kind of Minion job (which doesn't require arguments) but so far it is mainly related to cleanup.